securedrop/source_app/__init__.py

```diff
@@ -92,7 +96,7 @@ def handle_csrf_error(e):
     app.jinja_env.filters['filesizeformat'] = template_filters.filesizeformat
 
     for module in [main, info, api]:
-        app.register_blueprint(module.make_blueprint(config))
+        app.register_blueprint(module.make_blueprint(config))  # type: ignore
```

Comment:
Does this not return `None`?

Reply:
mypy/typing still can not handle module level typing that well as it sees the list as `List[ModuleType]` and `ModuleType` does not have that function `make_blueprint`.

Reply:
:+1: